src/main/java/controller/product/DeleteProductServlet.java

```diff
@@ -0,0 +1,40 @@
+package controller.product;
+
+import factory.ProductServiceFactory;
+import model.Product;
+import service.product.ProductService;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(value = "/deleteProduct")
+public class DeleteProductServlet extends HttpServlet {
+
+    private static final ProductService productService = ProductServiceFactory.getInstance();
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
```

Comment:
maybe better use one method to remove product?

Reply:
i cant. i need both methods, doGet for reference in users.jsp, doPost for changing in DB.